hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -43,6 +47,9 @@ objects:
           - key: api.openshift.com/environment
             operator: In
             values: ["staging", "integration"]
+          - key: api.openshift.com/legal-entity-id
+            operator: In
+            values: "${{TARGET_ORG}}"
```

Comment:
You need to remove the quotes here see [this](https://github.com/openshift/pagerduty-operator/blob/master/hack/olm-registry/olm-artifacts-template.yaml#L91) example.

Reply:
done!